src/operators.jl

```diff
@@ -172,9 +169,9 @@ function Base.:^(lhs::AbstractVariableRef, rhs::Integer)
         return one(GenericQuadExpr{Float64,variable_ref_type(lhs)})
     else
         error(
-            "Only exponents of 0, 1, or 2 are currently supported. Are you " *
-            "trying to build a nonlinear problem? Make sure you use " *
-            "@NLconstraint/@NLobjective.",
+            "Only integer exponents of 0, 1, or 2 are currently supported. " *
+            "Are you trying to build a nonlinear problem? Use the exponent " *
+            "$(Float64(rhs)) instead.",
```

Comment:
I find this message a bit obscure for the end user. Here, we do not know to what exactly `rhs` is referring to, unless we look at the arguments of this function. I would maybe clarify this, and give an example like: 
```
If you are trying to build a nonlinear problem, specify the exponent as a Float64
(e.g. `a^3` should be replaced by `a^3.0`). 
 
```

Reply:
Done.